relnotes/quickrun.feature.md

```diff
@@ -0,0 +1,23 @@
+### Helper to quickly spawn a task for scripting
+
+* `ocean.task.util.QuickRun`
+
+Utility intended for script-like usage:
+
+```D
+  int main ( )
+  {
+    initScheduler(SchedulerConfiguration.init);
```

Comment:
I would still argue for silently doing this as a default if the user doesn't do it, but it's not a big deal if you still disagree.

Reply:
Yeah I haven't changed my opinion on topic ;)